src/api/words/words.interface.ts

```diff
@@ -1,7 +1,10 @@
+import { LanguageCode } from 'iso-639-1'
```

Comment:
I think we should create 
GlobalLanguageCode within the global interface
and import there instead, so that those who are not familar with can use the code easier.
And modify right away, if the imported library is defacted.